File: java/Samples/Biometrics/VB/LatentFingerprintSampleVB/ExtractionSettingsForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms

Partial Public Class ExtractionSettingsForm
	Inherits Form
	#Region "Public constructor"

	Public Sub New()
		InitializeComponent()
	End Sub

	#End Region

	#Region "Public properties"

	Public Property QualityThreshold() As Byte
		Get
			Return Convert.ToByte(nudThreshold.Value)
		End Get
		Set(ByVal value As Byte)
			nudThreshold.Value = value
		End Set
	End Property

	#End Region

	#Region "Private events"

	Private Sub BtnDefaultThresholdClick(ByVal sender As Object, ByVal e As EventArgs) Handles btnDefaultThreshold.Click
		nudThreshold.Value = 39
	End Sub

	#End Region
End Class

File: java/Samples/Biometrics/CS/LatentFingerprintSampleCS/ExtractionSettingsForm.cs
using System;
using System.Windows.Forms;

namespace Neurotec.Samples
{
	public partial class ExtractionSettingsForm : Form
	{
		#region Public constructor

		public ExtractionSettingsForm()
		{
			InitializeComponent();
		}

		#endregion

		#region Public properties

		public byte QualityThreshold
		{
			get { return Convert.ToByte(nudThreshold.Value); }
			set { nudThreshold.Value = value; }
		}

		#endregion

		#region Private events

		private void BtnDefaultThresholdClick(object sender, EventArgs e)
		{
			nudThreshold.Value = 39;
		}

		#endregion
	}
}
